sharding-core/sharding-core-parse/sharding-core-parse-mysql/src/main/antlr4/imports/mysql/BaseRule.g4

```diff
@@ -262,6 +262,7 @@ expr
     | notOperator_ expr
     | LP_ expr RP_
     | booleanPrimary_
+    | predicate
```

Comment:
Why need add predicate here? It seems different with MySQL document.

Reply:
@terrymanu Just put predicate into above level.